bot/commands/utility/coinToss.js

```diff
@@ -0,0 +1,25 @@
+const source = require('rfr');
+const randomNumber = require('pure-random-number')
+
+
+const { sendMessage } = source('bot/utils/util');
+const logger = source('bot/utils/logger');
+
+function coinTossCommand(message) {
+
+    return randomNumber(1, 2)
+      .then(number => {
+          sendMessage(message.channel, `:coin: ${number === 1 ? 'heads' : 'tails'}`)}
+      )
+      .catch(err => logger.error('error tossing the coin:', err))
+}
+
+module.exports = {
+    args: false,
+    name: 'toss-coin',
```

Comment:
Can we use `!flip` instead of `toss-coin`?

Reply:
Yup, for some reason my mind blanked at what would be a good name last night.

Reply:
done